infra/helm/meshdb/values.yaml

```diff
@@ -56,7 +58,13 @@ nginx:
   port: 80
   podSecurityContext: {}
   securityContext: {}
-  resources: {}
+  resources:
+    limits:
+      cpu: 1
+      memory: 100Mi
+    requests:
+      cpu: 250m
+      memory: 30Mi
```

Comment:
This guy _definitely_ doesn't need more than 250mcore and like, 128Mi of RAM. If our web proxy is taking up more than that, we've either got wayyyy more users than we thought we did, or we have a performance problem.